### PR TITLE
Adds hint in ACI section to mention the architecture of your container.

### PR DESCRIPTION
**Pull request by jburns24**
Fixes cspell bug

### CODE REVIEW

File: .cspell.json [hunk 1]
```diff
@@ -73,7 +73,7 @@
         "pvcs",
         "Qube",
         "reate",
-        "SARIF"
+        "SARIF",
```
Comment (jburns24) [inline .cspell.json:76]: @rhoofard -- I missed in code review but wanted to point out the typo

File: docs/3-cloud-computing/3.3.5-aci.md [hunk 2]
```diff
@@ -49,6 +49,8 @@ EOF
 
 3. Build the Docker image locally.
 
+?> ACI architecture is x86. Make sure your container is compiled for x86. `docker inspect --format='{{.Os}}/{{.Architecture}}' <my_image>` should show `linux/amd64`
```
Comment (jburns24) [inline docs/3-cloud-computing/3.3.5-aci.md:52]: Liatrio issues M1/M2 macs and images compiled for arm were not working on ACI and giving strange errors

Comment (Wolftousen) [reply docs/3-cloud-computing/3.3.5-aci.md:52]: Add in a note about the `--platform linux/amd64` flag also?

Comment (jburns24) [reply docs/3-cloud-computing/3.3.5-aci.md:52]: Intentionally left this hint somewhat vague. As I want them to look into multi-architecture docker builds

**Review by Wolftousen — APPROVED** [review]

**Pull request merged**